### DISCUSSION TITLE
chaingreen synchronization

### DISCUSSION THREAD

**Discussion by marktoma96** [Chia-Network/chia-blockchain#8240] [Support]
I've been waiting for synchronization for more than 24 hours: is this normal?

**Comment by loppefaaret** [comment]
This is Chia, i'd suggest go asking Chaingreen about it

**Comment by BrandtH22** [comment]
@marktoma96 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
